tests/client/coll.rs

```diff
@@ -14,7 +12,7 @@ fn find_and_insert() {
 
     // Insert document
     let doc = doc! {
-        "title" => Bson::String("Jaws".to_owned())
+        "title" => (Bson::String("Jaws".to_owned()))
```

Comment:
Any way to work around having to put the parens here or can we use a different delimiter? Parenthesis are cool but the fact that parenthesis matter here makes the code a little more difficult to read. Might even be better to have an explicit `v!()` macro.